src/css/components/main.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  --main-color: #696969;
+  --main-color: rgb(245, 243, 243);
```

Comment:
You may want to coordinate your colors with the other pair. They had a hex color that was very similar to this, but I’m not sure if the values match exactly.